### PR TITLE
Ms new version reservia

### PR DESCRIPTION
**Pull request by Said2000-0ss**
new version but not finish, tomorrow is new day for finish the best version.

### CODE REVIEW

File: Reservia.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                </button>
+                <button class="pictureButton" type=submit>
+                    <img id="img3" src="Reservia_files/Romantic.png">
+                    </button>
+                    <button class="pictureButton" type=submit>
+                        <img id="img4" src="Reservia_files/Autorisedanimals.PNG">
+                        </button>
+      </div>
+      
+        <p ><img class="Informationpicture" src="Reservia_files/information.PNG"alt="search">
```
Comment (benjaminW78) [inline Reservia.html]: `Informationpicture` doesn't match the same pattern as the one in header or in buttons just above. is it normal? for a better readability you should work with the same naming convention when ever you write some CSS code.

example of not the same patterns: 
```
<button class="pictureButton" type=submit>
<img class="Informationpicture" src="Reservia_files/information.PNG"alt="search">
<header class="pageHead">
```

you have to choose one and stick to it :)

a very good article to look at with a lot of informations on best practices: https://hackernoon.com/best-practice-in-css-organisation-and-naming-conventions-4d103ujy

File: Reservia.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+<br><br>
+
+    <div class="test">
+
+        <section class="section1"><!--section 1 :  container of two boxes-->
+            
+            <div id="box1"><!-- one box-->
+                
+                <div id="sousbox1-1">
```
Comment (benjaminW78) [inline Reservia.html]: french css class, i thought we said it should be in plain english :) ?

File: Reservia.html [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+                    <figcaption class="figcaption1">Hotel chez Amina</figcaption>
+                   <p>Nuit à partir de <span>96 €</span></p>
+                    <p><img id="star" src="Reservia_files/star5.PNG" alt=""></p>
+                <!--</figure> -->
+                  
+                </div>
+            </div>
+      
+            
+            <div id="box2"><!-- Two box -->
```
Comment (benjaminW78) [inline Reservia.html]: what am i supposed to learn from this comment ?
because you are only repeating the information which is in the ID

File: Reservia.html [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+                <div id="sousbox2-3">
+                    <img id="hostimg9" class="imgbox2"src="Reservia_files/hostimg9.jpg" alt="">
+                    <figcaption class="figcaption2"id="idFigcaption2">Hotel tout bleu et blanc</figcaption>
+                </div>
+            </div>
+
+
+
+        </section>
+   <!--</br>
```
Comment (benjaminW78) [inline Reservia.html]: useless code, take time to remove everything that is not needed anymore. if not this will become legacy code and will become a nightmare to handle for the next developer to work on this page (of course in this case nobody else will work on it but hey that's a good habbit to take to remove deprecated code that is not used anymore)

**Review by benjaminW78 — CHANGES_REQUESTED** [review]
some changes to do before moving forward and merging this branch. please take the time to adress all my comments and commit the changes when you are done and let me know by answering to each comment individually when you fixed one of the things i pointed out. If you desagree to something let me know about it and let's discuss a possible solution.

answer in french if your not at ease with english. 
 every comments i made is only made to improve your code and make you ask the good questions on the path to develope your responsive web app. it's not a criticisme of your code but a discussion to enhance it 
 
 a good reminder about why Pull request are important just in case : https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

File: Reservia_files/style.css [hunk 5]
```diff
@@ -0,0 +1,620 @@
...
+background-color:#0065FC;
+width:20%;
+    height:35px;
+    border-top-right-radius: 12%;
+    border-bottom-right-radius:12%;
+    border:none;
+    grid-column-start: 3;
+
+}
+/*.fit-picture{
```
Comment (benjaminW78) [inline Reservia_files/style.css:99]: if it's useless remove it :)

File: Reservia_files/style.css [hunk 6]
```diff
@@ -0,0 +1,620 @@
...
+    grid-row-start: 3;
+    background-color:white;
+    border-radius: 2%;
+    margin-left: 2%;
+    margin-right: 2%;
+    margin-bottom: 2%;
+   
+    
+}
+/*.section1>#box1>.imgbox1{
```
Comment (benjaminW78) [inline Reservia_files/style.css:273]: to be removed

File: Reservia_files/style.css [hunk 7]
```diff
@@ -0,0 +1,620 @@
...
+    margin-bottom: 2%;
+}
+.section1>#box2>#sousbox2-2{
+    grid-row-start: 2;
+    background-color:white;
+    border-radius: 2%;
+    margin-left: 2%;
+    margin-right: 2%;
+    margin-bottom: 2%;
+    /*border:solid 1px;*/
```
Comment (benjaminW78) [inline Reservia_files/style.css:260]: to be removed

File: Reservia_files/style.css [hunk 8]
```diff
@@ -0,0 +1,620 @@
...
+.imgbox1{
+   
+    width: 94%;
+    height: 50%;
+    padding-top: 3%;
+    padding-left: 3%;
+    Padding-right:3%;
+    border-top-left-radius: 7%;
+    border-top-right-radius:7% ;
+   /* padding-top: 0px;
```
Comment (benjaminW78) [inline Reservia_files/style.css:375]: to be removed

File: Reservia_files/style.css [hunk 9]
```diff
@@ -0,0 +1,620 @@
...
+.imgbox2{
+    width: 50%;
+    height: 90%;
+    padding-top: 3%;
+    padding-left: 3%;
+    padding-bottom:3%;
+    border-top-left-radius: 10%;
+  
+    border-bottom-left-radius: 10%;
+
```
Comment (benjaminW78) [inline Reservia_files/style.css:391]: empty lines for nothing. to be removed

File: Reservia_files/style.css [hunk 10]
```diff
@@ -0,0 +1,620 @@
...
+    
+}
+
+.figcaption1{
+    padding-left: 3%;
+    font-weight: bold;
+
+}
+.figcaption2{
+
```
Comment (benjaminW78) [inline Reservia_files/style.css:403]: empty lines for nothing. to be removed

File: Reservia_files/style.css [hunk 11]
```diff
@@ -0,0 +1,620 @@
...
+    padding-right: 3%;
+    
+    
+
+}
+#idFigcaption1{
+    padding-right: 11%; 
+}
+#idFigcaption2{
+
```
Comment (benjaminW78) [inline Reservia_files/style.css:417]: empty lines for nothing. to be removed

File: Reservia_files/style.css [hunk 12]
```diff
@@ -0,0 +1,620 @@
...
+  #star{
+    padding-left: 1%;
+    width:30%;
+    height: 20%; 
+    padding-top:0%;
+ }
+ .section2{
+    display: grid;
+    background-color:white;
+
```
Comment (benjaminW78) [inline Reservia_files/style.css:442]: empty lines for nothing. to be removed

useless code commented, it should be removed instead.

File: Reservia_files/style.css [hunk 13]
```diff
@@ -0,0 +1,620 @@
...
+    grid-template-columns: 25% 25% 25% 25%;
+    /*grid-template-rows: 20% 20% 20% 20% 20%;*/
+    
+}
+
+.section2>#box1{
+    grid-column-start: 1;
+    background-color:white;
+    border-radius: 7%;
+   /* display:grid;
```
Comment (benjaminW78) [inline Reservia_files/style.css:452]: to be removed if not usefull.

File: Reservia_files/style.css [hunk 14]
```diff
@@ -0,0 +1,620 @@
...
+    padding-left: 4%;
+    padding-right: 4%;
+    padding-top:15%;
+    padding-bottom: 4%;
+    border-radius: 2%;
+    display:grid;
+    grid-template-rows: 30% 30% 30% ;
+    margin-right: 5%;
+}
+.section1>#box1>#sousbox1-1{
```
Comment (benjaminW78) [inline Reservia_files/style.css:182]: the selectors bellow
are using almost the same css properties! you should try to create a class that include margins-tops etc... =2% and background color white and finally use ids only for grid-column position.

without even changing your html you could do following for applying the margins and background color.
```suggestion
#box1>[id^="sousbox"]{
    background-color:white;    
    grid-column: 1;
    border-radius: 2%;
    margin-top: 2%;    
    margin-left: 2%;    
    margin-right: 2%;    
    margin-bottom: 2%;
}
```
Some more details about this kind of selectors  ( it's a must read) https://developer.mozilla.org/fr/docs/Web/CSS/Attribute_selectors


and then in your #sousbox1-1 you can just choose the column number but i would advise you to get ride telling which div is which column and to use automatic placement https://developer.mozilla.org/fr/docs/Web/CSS/CSS_Grid_Layout/Auto-placement_in_CSS_Grid_Layout

File: Reservia_files/style.css [hunk 15]
```diff
@@ -0,0 +1,620 @@
+
```
Comment (benjaminW78) [inline Reservia_files/style.css:1]: didn't we say this file should be splet in many smallers one ? to improve readability and maintenance?

**Review by benjaminW78 — CHANGES_REQUESTED** [review]

File: Reservia.html (hunk 1 above)
Comment (Said2000-0ss) [reply Reservia.html]: I changed InformationPicture to informationPicture in order to respect the naming conventions.

File: Reservia.html (hunk 2 above)
Comment (Said2000-0ss) [reply Reservia.html]: I modified sousbox by putting the French word in English by: "sub-box"

File: Reservia.html (hunk 3 above)
Comment (Said2000-0ss) [reply Reservia.html]: Indeed, I repeat this information, it allows me to see more clearly in my source code.

File: Reservia.html (hunk 4 above)
Comment (Said2000-0ss) [reply Reservia.html]: the useless code has been erased in order to have a clean code.

File: Reservia_files/style.css (hunk 5 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:99]: it's useless, so I deleted it.

File: Reservia_files/style.css (hunk 6 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:273]: I deleted this piece of code.

File: Reservia_files/style.css (hunk 7 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:260]: I deleted this piece of code.

File: Reservia_files/style.css (hunk 8 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:375]: I deleted this piece of code.

File: Reservia_files/style.css (hunk 9 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:391]: I deleted the empty lines.

File: Reservia_files/style.css (hunk 10 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:403]: Empty lines have been removed.

File: Reservia_files/style.css (hunk 11 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:417]: Empty lines have been removed.

File: Reservia_files/style.css (hunk 12 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:442]: Empty lines have been removed.

File: Reservia_files/style.css (hunk 13 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:452]: I removed this unnecessary code.

File: Reservia_files/style.css (hunk 15 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:1]: Indeed, we said that this file should be broken down into several small parts. Here is the new file. Cordially.

Comment (Said2000-0ss) [reply Reservia_files/style.css:1]: I proceeded to the factorization of the code, according to the repetition of the code.

File: Reservia_files/style.css (hunk 14 above)
Comment (Said2000-0ss) [reply Reservia_files/style.css:182]: I proceeded to the factorization of the code, according to the repetition of the code.

**Comment by Said2000-0ss** [comment]
Hello, I made all of the changes, if you can take a look, thank you :)

File: Reservia.html [hunk 16]
```diff
@@ -0,0 +1,215 @@
...
+        </nav>
+        <a class="pageHead__itemRegister" href="page_register.html">S'inscrire</a>
+    </header>
+<!------------------------------------------------------------------END HEADER ------------------------------------------------------------>
+
+
+
+
+<!------------------------------------------------------------------- NAVIGATION ---------------------------------------------------------->
+    <h2 id="h2navigation">Trouvez votre hébergement pour des vacances de reves</h2>
```
Comment (benjaminW78) [inline Reservia.html:33]: this h2 is not having the same style as the one on the other sections ?

File: Reservia.html (hunk 3 above)
Comment (benjaminW78) [reply Reservia.html]: yes but it's not good practice to keep useless comments in code so keep in mind you will have to remove it by the end of your project.

File: Reservia_files/css/activities.css [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+	width:302px;
+	height:445px;
+    grid-column-start: 4;
+    border-radius: 7px;
+    margin-left: 42px;
+    margin-right: 54px;
+    display:grid;
+    grid-template-rows: 50% 50%;   
+}
+				.section2>#box4>#box4-1{
```
Comment (benjaminW78) [inline Reservia_files/css/activities.css:68]: wrong indentation

**Comment by benjaminW78** [comment]
last comment, you stylised everything with ID's that's not a good practice at all.  have a look at this article and next PR you are gonna have to work on that :)

File: Reservia.html (hunk 16 above)
Comment (Said2000-0ss) [reply Reservia.html:33]: new indentation performed

File: Reservia_files/css/activities.css (hunk 17 above)
Comment (Said2000-0ss) [reply Reservia_files/css/activities.css:68]: new indentation performed

File: Reservia.html (hunk 16 above)
Comment (Said2000-0ss) [reply Reservia.html:33]: I just transformed the H2 into H1

Comment (benjaminW78) [reply Reservia.html:33]: don't forget to push your commit because i don't see the changes yet.

**Comment by benjaminW78** [comment]
@Said2000-0ss  you can merge if you are happy with that already because every comments are solved

**Comment by Said2000-0ss** [comment]
Bonjour Benjamin,
je te soumets la pull request comme prévu.
La vidéo sera faite ce weekend , tu l'auras pour lundi.
Bon weekend à toi.
à très bientôt.
Saïd MANSSOURI

+33 (0)6 18 46 39 49
***@***.***


<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Said2000-0ss** [comment]
PS :  je t'adresse le lien github (même si je pense que tu dois l'avoir ;-)
)
https://github.com/Said2000-0ss/OpenClassRooms-Projet-2.git


Saïd MANSSOURI

+33 (0)6 18 46 39 49
***@***.***


<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Le ven. 27 août 2021 à 17:40, Said Manssouri ***@***.***> a
écrit :

**Comment by Said2000-0ss** [comment]
Salut Benjamin ,
je t'adresse comme prévu le lien youtube :
https://youtu.be/LhPDvOBznK4
Bonne fin de soirée à toi.
 A très bientôt.
Saïd MANSSOURI

+33 (0)6 18 46 39 49
***@***.***


<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Le ven. 27 août 2021 à 17:43, Said Manssouri ***@***.***> a
écrit :
